From 824bbf4b42fb086908a9fb6017cf46a830faf13f Mon Sep 17 00:00:00 2001
From: Myy <myy@miouyouyou.fr>
Date: Wed, 19 Jul 2017 00:26:17 +0000
Subject: [PATCH] Remove the dependency to the clk_mali symbol.

Turns out that defining a new clock symbol is not useful.

See https://irclog.whitequark.org/linux-rockchip/2017-07-18 for
informations.

Signed-off-by: Myy <myy@miouyouyou.fr>
---
 drivers/gpu/arm/midgard/mali_kbase_core_linux.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/arm/midgard/mali_kbase_core_linux.c b/drivers/gpu/arm/midgard/mali_kbase_core_linux.c
index 9cc65d2..db3ac6a 100644
--- a/drivers/gpu/arm/midgard/mali_kbase_core_linux.c
+++ b/drivers/gpu/arm/midgard/mali_kbase_core_linux.c
@@ -3793,7 +3793,7 @@ static int power_control_init(struct platform_device *pdev)
 	}
 #endif /* LINUX_VERSION_CODE >= 3, 12, 0 */
 
-	kbdev->clock = clk_get(kbdev->dev, "clk_mali");
+	kbdev->clock = clk_get(kbdev->dev, NULL);
 	if (IS_ERR_OR_NULL(kbdev->clock)) {
 		err = PTR_ERR(kbdev->clock);
 		kbdev->clock = NULL;
-- 
2.10.2

